build-system/tasks/visual-diff/dev-mode.js

```diff
@@ -0,0 +1,200 @@
+'use strict';
+
+const inquirer = require('inquirer');
+const path = require('path');
+const puppeteer = require('puppeteer'); // eslint-disable-line no-unused-vars
```

Comment:
Since this is purely for typing, any chance it can be replaced by something inside https://www.npmjs.com/package/@types/puppeteer?

(The disable directive is not an actual concern, so feel free to ignore.)

Reply:
This throws `Cannot import type declaration files. Consider importing 'puppeteer' instead of '@types/puppeteer'.ts(6137)`

Another option is to use the `import(...).Type` syntax which I personally find unreadable:
```js
/**
 * @param {!import('puppeteer').Browser} browser a Puppeteer controlled browser.
 */
```

lmk if you have a strong opinion

Reply:
Cool, leaving this as is SGTM.